lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -0,0 +1,782 @@
+  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-Identifiers"));
+  if (identifiers > 0)
+  {
+    if (identifiers > size)
+      identifiers = size;
+    Description annotations(size);
+    Sample dataWithIndex(size, 2);
+    for(UnsignedInteger i = 0; i < size; ++i)
+    {
+      dataWithIndex(i, 0) = std::abs(residuals(i, 0));
+      dataWithIndex(i, 1) = i;
+    }
+    const Sample sortedData(dataWithIndex.sortAccordingToAComponent(0));
+    Description positions(size, "top");
+    for(UnsignedInteger i = 0; i < identifiers; ++i)
+    {
+      const UnsignedInteger index = sortedData(size - 1 - i, 1);
+      annotations[index] = (OSS() << index + 1);
+      if (residuals(index, 0) < 0.0)
+        positions[index] = "top";
+      else
+        positions[index] = "bottom";
+    }
+    Text text(dataFull, annotations, "bottom");
+    text.setColor("red");
+    text.setTextPositions(positions);
+    graph.add(text);
+  }
+  return graph;
+}
+
+/* [2] a Scale-Location plot of sqrt(| residuals |) versus fitted values */
+Graph LinearModelAnalysis::drawScaleLocation() const
+{
+  const Sample inputData(linearModelResult_.getInputSample());
+  const Function metamodel(linearModelResult_.getMetaModel());
+  const Sample fitted(metamodel(inputData));
+  const Sample stdresiduals(getStandardizedResiduals());
+  const UnsignedInteger size(fitted.getSize());
+  Sample dataFull(fitted);
+  Sample sqrtstdresiduals(size, 1);
+  for(UnsignedInteger i = 0; i < size; ++i)
+  {
+    sqrtstdresiduals(i, 0) = std::sqrt(std::abs(stdresiduals(i, 0)));
+  }
+  dataFull.stack(sqrtstdresiduals);
+  Graph graph("Scale-Location", "Fitted values", "|Std. residuals|^0.5", true, "topright");
+  Cloud cloud(dataFull, "black", "fcircle");
+  graph.add(cloud);
+  // Add point identifiers for worst standardized residuals
+  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-Identifiers"));
+  if (identifiers > 0)
+  {
+    if (identifiers > size)
+      identifiers = size;
+    Description annotations(size);
+    Sample dataWithIndex(size, 2);
+    for(UnsignedInteger i = 0; i < size; ++i)
+    {
+      dataWithIndex(i, 0) = std::abs(stdresiduals(i, 0));
+      dataWithIndex(i, 1) = i;
+    }
+    const Sample sortedData(dataWithIndex.sortAccordingToAComponent(0));
+    Description positions(size, "top");
+    for(UnsignedInteger i = 0; i < identifiers; ++i)
+    {
+      const UnsignedInteger index = sortedData(size - 1 - i, 1);
+      annotations[index] = (OSS() << index + 1);
+      if (stdresiduals(index, 0) < 0.0)
+        positions[index] = "top";
+      else
+        positions[index] = "bottom";
+    }
+    Text text(dataFull, annotations, "bottom");
+    text.setColor("red");
+    text.setTextPositions(positions);
+    graph.add(text);
+  }
+  return graph;
+}
+
+/* [3] a Normal quantiles-quantiles plot of standardized residuals */
+Graph LinearModelAnalysis::drawQQplot() const
```

Comment:
The particular points are : 
 * the bissectrice (using quantiles of level 0.75 and 0.25 instead of min/max)
 * The identification of the worst points --> Adding a new drawable

Maybe future step is updating the VisualTest if the first item is of interest such as the specific implementation in LinearModelAnalysis concerns the identification of the worst residuals